central/cve/info/datastore/datastore.go

```diff
@@ -0,0 +1,40 @@
+package datastore
```

Comment:
Is this just for images or for everything?  If just for image, move it to `centra/cve/image/info`